checks.d/etcd.py

```diff
@@ -68,6 +68,17 @@ def check(self, instance):
         # Load values from the instance config
         url = instance['url']
         instance_tags = instance.get('tags', [])
+
+        ssl_params = {
+            'ssl': instance.get('ssl', False),
```

Comment:
You probably want to use `_is_affirmative` see https://github.com/DataDog/dd-agent/commit/0e862f5f7f16f5fc1466f4b8ebff1b4e44cac64e.